server/service/osquery_utils/queries.go

```diff
@@ -750,6 +758,32 @@ func discoveryTable(tableName string) string {
 	return fmt.Sprintf("SELECT 1 FROM osquery_registry WHERE active = true AND registry = 'table' AND name = '%s';", tableName)
 }
 
+func macOSBundleIDExistsQuery(appName string) string {
+	return fmt.Sprintf("SELECT 1 FROM apps WHERE bundle_identifier = '%s' LIMIT 1", appName)
+}
+
+// multiCondition generates a SQL query that returns 1 if all subqueries return 1, otherwise 0.
+// subqueries should be a list of SQL queries that return 1 if a condition is met, otherwise 0.
+func generateSQLForAllExists(subqueries ...string) string {
+	if len(subqueries) == 0 {
+		return "SELECT 0;" // Return 0 if no subqueries provided
```

Comment:
`SELECT 0;` is actually a success result when it comes to discovery queries (result vs no result like policies).

Reply:
ahh, good catch.  I added a LIMIT 0 here.